src/main/java/uk/gov/justice/probation/courtcasematcher/model/domain/Defendant.java

```diff
@@ -43,6 +43,7 @@ public class Defendant {
 
     private Offender offender;
     private Boolean confirmedOffender;
+    private String personId;
```

Comment:
This also needs to be added to the rest client DTOs and mapped to and from.

Reply:
The Pact tests should also be updated to assert that personId is returned from court-case-service. The changes will be very similar to those from [this PR ](https://github.com/ministryofjustice/court-case-matcher/pull/530).